srcpkgs/unar/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'unar'
+pkgname=unar
+version=1.10.1
+revision=1
+archs="i686 x86_64" # gnustep broken on musl
+wrksrc="kosovan-theunarchiver-1534a5cf6900"
+makedepends="gnustep-make gnustep-base-devel gcc-objc bzip2-devel"
```

Comment:
`gnustep-make` shouldn't be necessary. And I think `gcc-objc` should be in `hostmakedepends`